recipe/meta.yaml

```diff
@@ -67,6 +67,7 @@ outputs:
 
   - name: gap-defaults
     script: install-defaults.sh
+    # ... and why does this "output" not need any requirements, when the one before and the one after apparently do need them?
     requirements:
       host:
         - {{ pin_subpackage('gap-core', exact=True) }}
```

Comment:
This has gap-core as a dependency so recursively all its dependencies.

Reply:
But `gap` below also has `gap-core` as dependency, yet also repeats some of them?